core/src/test/java/org/infinispan/distribution/rehash/ConcurrentJoinTest.java

```diff
@@ -26,7 +26,7 @@ void performRehashEvent(boolean offline) throws Exception {
       joiners = new CopyOnWriteArrayList<Cache<Object, String>>(new Cache[NUM_JOINERS]);
 
       for (int i = 0; i < NUM_JOINERS; i++) {
-         EmbeddedCacheManager joinerManager = addClusterEnabledCacheManager(new TransportFlags().withFD(false));
+         EmbeddedCacheManager joinerManager = addClusterEnabledCacheManager(new TransportFlags().withFD(false).withPortRange(i));
```

Comment:
You should do the same in `StateTransferFunctionalTest.testConcurrentStateTransfer`.

Although I'd rather see this solved in JGroups :)